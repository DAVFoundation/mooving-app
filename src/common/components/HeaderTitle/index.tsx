import React from 'react';
import { View, Text, StyleSheet } from 'react-native';
import { customSizes } from '../../../common/styles';

export function HeaderTitle(props: any) {
  return (
    <View style = {styles.headerTitle}>
      <Text style={styles.headerTitleText}>{props.title}</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  headerTitle: {
    borderRadius: customSizes.main / 2,
    width: customSizes.main * 3,
    height: customSizes.space * 2,
    backgroundColor: '#fff',
    justifyContent: 'space-around',
  },
  headerTitleText: {
    color: '#000',
    textAlign: 'center',
    fontSize: 15,
  },
});
