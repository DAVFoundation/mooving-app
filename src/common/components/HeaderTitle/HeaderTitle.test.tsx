import React from 'react';
import renderer from 'react-test-renderer';
import { HeaderTitle } from '../HeaderTitle';

it('renders correctly with defaults', () => {
  const view = renderer
    .create(<HeaderTitle title='Hello World!'/>)
    .toJSON();
  expect(view).toMatchSnapshot();
});
