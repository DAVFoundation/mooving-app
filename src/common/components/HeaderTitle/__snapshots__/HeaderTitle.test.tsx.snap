// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with defaults 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderRadius": 27.5,
      "height": 36.666666666666664,
      "justifyContent": "space-around",
      "width": 165,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#000",
        "fontSize": 15,
        "textAlign": "center",
      }
    }
  >
    Hello World!
  </Text>
</View>
`;
